api/php/services/RestaurantService.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+include "php/Dbgateways/RestaurantDbGateway.php";
+
+class RestaurantService {
+    private $restaurantdbGateway;
+
+    public function __construct() {
+        $this->restaurantDbGateway = new RestaurantDbGateway();
+    }
+
+    public function processReview (string $restaurantName, string $restaurantAddress, string $restaurantPhone,
+        string $restaurantWebsite, string $review, int $deliciousnessScore, int $serviceScore, int $experienceScore, int $pricingScore, float $pricingValue,
+        array $images) {
+        if (strlen($review) == 0) {
+            return "Review cannot be blank";
+        } else {
+            $restaurantId = $this->restaurantDbGateway->validateRestaurant($restaurantName, $restaurantAddress, $restaurantPhone, $restaurantWebsite);
```

Comment:
$restaurantId = $this->restaurantDbGateway->validateRestaurant($restaurantName, $restaurantAddress, $restaurantPhone, $restaurantWebsite);

RestaurantDbGateway is calling method validateRestaurant, we can rename this method because it is a little misleading. Validate is ensuring the restaurant exists in our database table but the return value is being stored in restaurantId variable. We should create a seperate method "getRestaurantId" which calls "validateRestaurant". This makes the code more efficent, readable, and self-documenting.